Base64 images are overriden by URL images

Using 2.16.15-beta (and some earlier betas), any binary `Image item` info in sitemap is replaced by a single URL-linked image after some milliseconds, in my case a webcam which is used on initial sitemap view as 1st image. `Image item` image is shown briefly for e.g. 1/10s, then replaced by webcam. Tapping it results in app crash when image is highres. In case image item is small (e.g. OWM weather condition icon), tapping false webcam image shows expected weather condition icon fullscreen.
Images resulting from direct URL access are shown well and without change to other image.
Identical behavior in portrait and landscape orientation.
Actual released app doesn't present that issue - but also doesn't offer fullscreen image yet.
Reduced to a test sitemap, some log sniplet, hope it's enough:
```
-----------------------
Device information
Model: SM-G991B
Manufacturer: samsung
Brand: samsung
Device: o1s
Product: o1sxeea
OS: 11
Display: 1080x2320, 2.625 density
Data usage policy: DataUsagePolicy(canDoLargeTransfers=true, loadIconsWithState=true, autoPlayVideos=true, canDoRefreshes=true), data saver: 1, battery saver: false
-----------------------
...

04-16 10:11:52.611 12387 12387 D PageConnectionHolderFragment: Updated page data for URL https://<openhab-local-address-1. openHAB 3>:8443/rest/sitemaps/passattest/passattest (5 widgets)
04-16 10:11:52.611 12387 12387 D PageConnectionHolderFragment: Widget 1: Widget(id=00, parentId=null, rawLabel=Passat Details, icon=IconResource(icon=frame, isOh2=true, customState=), state=null, type=Frame, url=null, item=null, linkedPage=null, mappings=[], encoding=null, iconColor=null, labelColor=null, valueColor=null, refresh=0, minValue=0.0, maxValue=100.0, step=1.0, period=D, service=, legend=null, switchSupport=false, height=0, visibility=true)
04-16 10:11:52.634 12387 12387 D PageConnectionHolderFragment: Widget 2: Widget(id=0000, parentId=00, rawLabel=Passat Bild, icon=IconResource(icon=car, isOh2=true, customState=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAB4AAAAQ4CAYAAADo08FDAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAAABmJLR0QA/wD/AP+gvaeTAACAAElEQVR42uz9WaxsWX7f+f3WHmM4050y7705VFaxpswqslikOBUpUhTJllpiq9UTbakluYUGWmg3YBiG/WIbaPlBD4aNhl/lx4btBhqNfnADbjQMQ6INDS2xisUqFqsqq7JyzrzzmWLcox/WXmuvvSPOzSyOWcnvB7h5z4kTEf8dO3bEuRm//f8vCQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA/FkzxlCf+tSnPvWpT33qU5/61Kc+9alPfepTn/rUpz71qU996lOf+tSnPvU/BvUBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMBHlzGG+tSnPvWpT33qU5/61Kc+9alPfepTn/rUpz71qU996lOf+tSnPvWp/zGoDwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD4aDHGUJ/61Kc+9alPfepTn/rUpz71qU996lOf+tSnPvWpT33qU5/61Kc+9T8G9QEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAwEeXMYb61Kc+9alPfepTn/rUpz71qU996lOf+tSnPvWpT33qU5/61Kc+9an/MagPAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAPhoMcZQn/rUpz71qU996lOf+tSnPvWpT33qU5/61Kc+9alPfepTn/rUpz71Pwb1AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAR5cxhvrUpz71qU996lOf+tSnPvWpT33qU5/61Kc+9alPfepTn/rUpz71qf8xqA8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA+GgxxlCf+tSnPvWpT33qU5/61Kc+9alPfepTn/rUpz71qU996lOf+tSnPvU/BvUBAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAMBHlzGG+tSnPvWpT33qU5/61Kc+9alPfepTn/rUpz71qU996lOf+tSnPvWp/zGoDwAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAD4aDHGUJ/61Kc+9alPfepTn/rUpz71qU996lOf+tSnPvWpT33qU5/61Kc+9T8G9QEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAwEeXMYb61Kc+9alPfepTn/rUpz71qU996lOf+tSnPvWpT33qU5/61Kc+9an/MagPAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAPhoMcZQn/rUpz71qU996lOf+tSnPvWpT33qU5/61Kc+9alPfepTn/rUpz71Pwb1AQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAADAR5cxhvrUpz71qU996lOf+tSnPvWpT33qU5/61Kc+9alPfepTn/rUpz71qf8xqA8AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA
04-16 10:11:52.634 12387 12387 D PageConnectionHolderFragment: Widget 3: Widget(id=0001, parentId=00, rawLabel=Trip Langzeit [- km], icon=IconResource(icon=odometer, isOh2=true, customState=), state=null, type=Text, url=null, item=Item(name=PassatTotalDistance, label=Kilometerstand, category=odometer, type=NumberWithDimension, groupType=None, link=https://<openhab-local-address-1. openHAB 3>:8443/rest/items/PassatTotalDistance, readOnly=true, members=[], options=[], state=null, tags=[]), linkedPage=LinkedPage(id=0001, title=Trip Langzeit , icon=IconResource(icon=odometer, isOh2=true, customState=null), link=https://<openhab-local-address-1. openHAB 3>:8443/rest/sitemaps/passattest/0001), mappings=[], encoding=null, iconColor=null, labelColor=null, valueColor=null, refresh=0, minValue=0.0, maxValue=100.0, step=1.0, period=D, service=, legend=null, switchSupport=false, height=0, visibility=true)
04-16 10:11:52.635 12387 12387 D PageConnectionHolderFragment: Widget 4: Widget(id=0002, parentId=00, rawLabel=Diesel [23 %], icon=IconResource(icon=sewerage, isOh2=true, customState=23), state=ParsedState(asString=23, asBoolean=true, asNumber=23 %, asHsv=null, asBrightness=null, asLocation=null), type=Text, url=null, item=Item(name=PassatFuelLevel, label=Diesel Füllstand, category=sewerage, type=Number, groupType=None, link=https://<openhab-local-address-1. openHAB 3>:8443/rest/items/PassatFuelLevel, readOnly=true, members=[], options=[], state=ParsedState(asString=23, asBoolean=true, asNumber=23 %, asHsv=null, asBrightness=null, asLocation=null), tags=[]), linkedPage=LinkedPage(id=0002, title=Diesel , icon=IconResource(icon=sewerage, isOh2=true, customState=null), link=https://<openhab-local-address-1. openHAB 3>:8443/rest/sitemaps/passattest/0002), mappings=[], encoding=null, iconColor=null, labelColor=null, valueColor=null, refresh=0, minValue=0.0, maxValue=100.0, step=1.0, period=D, service=, legend=null, switchSupport=false, height=0, visibility=true)
04-16 10:11:52.635 12387 12387 D PageConnectionHolderFragment: Widget 5: Widget(id=0003, parentId=00, rawLabel=, icon=IconResource(icon=map, isOh2=true, customState=xx.nnnnnn,yy.nnnnnn), state=ParsedState(asString=xx.nnnnnn,yy.nnnnnn, asBoolean=false, asNumber=null, asHsv=null, asBrightness=null, asLocation=Location[openhab xx,nnnnnn,yy,nnnnnn hAcc=??? et=?!? vAcc=??? sAcc=??? bAcc=???]), type=Mapview, url=null, item=Item(name=Passat_Location, label=Ort, category=map, type=Location, groupType=None, link=https://<openhab-local-address-1. openHAB 3>:8443/rest/items/Passat_Location, readOnly=true, members=[], options=[], state=ParsedState(asString=xx.nnnnnn,yy.nnnnnn, asBoolean=false, asNumber=null, asHsv=null, asBrightness=null, asLocation=Location[openhab xx,nnnnnn,yy,nnnnnn hAcc=??? et=?!? vAcc=??? sAcc=??? bAcc=???]), tags=[]), linkedPage=null, mappings=[], encoding=null, iconColor=null, labelColor=null, valueColor=null, refresh=0, minValue=0.0, maxValue=100.0, step=1.0, period=D, service=, legend=null, switchSupport=false, height=5, visibility=true)
04-16 10:11:52.635 12387 12387 D ContentController: Got update for URL https://<openhab-local-address-1. openHAB 3>:8443/rest/sitemaps/passattest/passattest, pending []
04-16 10:11:53.888 12387 12387 I ViewRootImpl@7550a32[MainActivity]: ViewPostIme pointer 0
04-16 10:11:53.925 12387 12387 I ViewRootImpl@7550a32[MainActivity]: ViewPostIme pointer 1
04-16 10:11:53.939 12387 12387 E JavaBinder: !!! FAILED BINDER TRANSACTION !!!  (parcel size = 5510816)
04-16 10:11:53.939 12387 12387 D AndroidRuntime: Shutting down VM
--------- beginning of crash
04-16 10:11:53.940 12387 12387 E AndroidRuntime: FATAL EXCEPTION: main
04-16 10:11:53.940 12387 12387 E AndroidRuntime: Process: org.openhab.habdroid.beta, PID: 12387
04-16 10:11:53.940 12387 12387 E AndroidRuntime: java.lang.RuntimeException: Failure from system
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.app.Instrumentation.execStartActivity(Instrumentation.java:1729)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.app.Activity.startActivityForResult(Activity.java:5376)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at androidx.fragment.app.FragmentActivity.startActivityForResult(FragmentActivity.java:675)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.app.Activity.startActivityForResult(Activity.java:5334)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at androidx.fragment.app.FragmentActivity.startActivityForResult(FragmentActivity.java:662)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.app.Activity.startActivity(Activity.java:5720)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.app.Activity.startActivity(Activity.java:5673)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at org.openhab.habdroid.ui.WidgetAdapter$ImageViewHolder.onClick(WidgetAdapter.kt:705)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.view.View.performClick(View.java:8160)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.view.View.performClickInternal(View.java:8137)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.view.View.access$3700(View.java:888)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.view.View$PerformClick.run(View.java:30236)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.os.Handler.handleCallback(Handler.java:938)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.os.Handler.dispatchMessage(Handler.java:99)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.os.Looper.loop(Looper.java:246)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.app.ActivityThread.main(ActivityThread.java:8462)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at java.lang.reflect.Method.invoke(Native Method)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:596)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1130)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: Caused by: android.os.TransactionTooLargeException: data parcel size 5510816 bytes
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.os.BinderProxy.transactNative(Native Method)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.os.BinderProxy.transact(BinderProxy.java:605)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.app.IActivityTaskManager$Stub$Proxy.startActivity(IActivityTaskManager.java:4221)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: at android.app.Instrumentation.execStartActivity(Instrumentation.java:1723)
04-16 10:11:53.940 12387 12387 E AndroidRuntime: ... 18 more
04-16 10:11:53.987 12387 14136 D TransportRuntime.SQLiteEventStore: Storing event with priority=HIGHEST, name=FIREBASE_CRASHLYTICS_REPORT for destination cct
04-16 10:11:53.989 12387 14136 D TransportRuntime.JobInfoScheduler: Scheduling upload for context TransportContext(cct, HIGHEST, MSRodHRwczovL2NyYXNobHl0aWNzcmVwb3J0cy1wYS5nb29nbGVhcGlzLmNvbS92MS9maXJlbG9nL2xlZ2FjeS9iYXRjaGxvZ1xBSXphU3lCcnBTWVQ0RkZMMDlyZUhKaTZIOUZZZGVpU25VVE92Mk0=) with jobId=-560122990 in 18179ms(Backend next call timestamp 1618560732168). Attempt 1
04-16 10:12:04.155 12171 12171 E b.habdroid.bet: Not starting debugger since process cannot load the jdwp agent.
04-16 10:12:04.157 12171 12171 D ActivityThread: setConscryptValidator
04-16 10:12:04.157 12171 12171 D ActivityThread: setConscryptValidator - put
--------- beginning of system
04-16 10:12:04.165 12171 12171 D ActivityThread: handleBindApplication()++ app=org.openhab.habdroid.beta
04-16 10:12:04.172 12171 12171 I b.habdroid.bet: The ClassLoaderContext is a special shared library.
04-16 10:12:04.184 12171 12171 W b.habdroid.bet: JIT profile information will not be recorded: profile file does not exist.
04-16 10:12:04.184 12171 12171 W b.habdroid.bet: JIT profile information will not be recorded: profile file does not exist.
04-16 10:12:04.187 12171 12171 D ActivityThread: handleBindApplication() -- skipGraphicsSupport=false
04-16 10:12:04.188 12171 12171 D ActivityThread: ActivityThread::handleMakeApplication() data=AppBindData{appInfo=ApplicationInfo{97dccc2 org.openhab.habdroid.beta}} appContext=android.app.ContextImpl@eff7ad3 appContext.mOpPackageName=org.openhab.habdroid.beta appContext.mBasePackageName=org.openhab.habdroid.beta appContext.mPackageInfo=android.app.LoadedApk@c65c610 Caller=android.app.ActivityThread.handleBindApplication:7411 android.app.ActivityThread.access$1500:301 android.app.ActivityThread$H.handleMessage:2130 android.os.Handler.dispatchMessage:106 android.os.Looper.loop:246
04-16 10:12:04.188 12171 12171 D LoadedApk: LoadedApk::makeApplication() appContext=android.app.ContextImpl@e3cf12f appContext.mOpPackageName=org.openhab.habdroid.beta appContext.mBasePackageName=org.openhab.habdroid.beta appContext.mPackageInfo=android.app.LoadedApk@c65c610
04-16 10:12:04.188 12171 12171 D NetworkSecurityConfig: Using Network Security Config from resource network_security_config debugBuild: false
04-16 10:12:04.192 12171 12171 D NetworkSecurityConfig: Using Network Security Config from resource network_security_config debugBuild: false
04-16 10:12:04.193 12171 12171 I MultiDex: VM with version 2.1.0 has multidex support
04-16 10:12:04.193 12171 12171 I MultiDex: Installing application
04-16 10:12:04.193 12171 12171 I MultiDex: VM has multidex support, MultiDex support library is disabled.
04-16 10:12:04.194 12171 12171 W ComponentDiscovery: Class com.google.firebase.dynamicloading.DynamicLoadingRegistrar is not an found.
04-16 10:12:04.195 12171 12171 I FirebaseApp: Device unlocked: initializing all Firebase APIs for app [DEFAULT]
04-16 10:12:04.197 12171 12171 I FirebaseCrashlytics: Initializing Firebase Crashlytics 17.4.1
04-16 10:12:04.205 12171 12171 I FirebaseInitProvider: FirebaseApp initialization successful
04-16 10:12:04.229 12171 12171 I TetheringManager: registerTetheringEventCallback:org.openhab.habdroid.beta
04-16 10:12:04.229 12171 12171 D ConnectivityManager: ConnectivityManager() mContext=org.openhab.habdroid.core.OpenHabApplication@1b94d7c getOpPackageName()=org.openhab.habdroid.beta getBasePackageName()=org.openhab.habdroid.beta getPackageName()=org.openhab.habdroid.beta
04-16 10:12:04.233 12171 12171 D CrashReportingHelper: Crashlytics status: isDebug false, isOutdated false
04-16 10:12:04.237 12171 14444 D TransportRuntime.JobInfoScheduler: Upload for context TransportContext(cct, HIGHEST, MSRodHRwczovL2NyYXNobHl0aWNzcmVwb3J0cy1wYS5nb29nbGVhcGlzLmNvbS92MS9maXJlbG9nL2xlZ2FjeS9iYXRjaGxvZ1xBSXphU3lCcnBTWVQ0RkZMMDlyZUhKaTZIOUZZZGVpU25VVE92Mk0=) is already scheduled. Returning...
04-16 10:12:04.255 12171 12171 I EngineFactory: Provider GmsCore_OpenSSL not available
04-16 10:12:04.308 12171 12171 D MainActivity: onCreate()
...
```

_Originally posted by @cwihne in https://github.com/openhab/openhab-android/issues/2483#issuecomment-821020687_

--- a/mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt
+++ b/mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt
@@ -103,6 +103,7 @@ class WidgetImageView constructor(context: Context, attrs: AttributeSet?) : AppC
     }
 
     fun setBase64EncodedImage(base64: String) {
+        prepareForNonHttpImage()
         val data = Base64.decode(base64, Base64.DEFAULT)
         val bitmap = BitmapFactory.decodeByteArray(data, 0, data.size)
 

